src/MSBuild/XMake.cs

```diff
@@ -1749,6 +1749,16 @@ private static List<BuildManager.DeferredBuildMessage> GetMessagesToLogInBuildLo
                     MessageImportance.Low),
             };
 
+            if (NativeMethodsShared.IsWindows)
+            {
+                messages.Add(
+                    new BuildManager.DeferredBuildMessage(
```

Comment:
This is a good place to put this, but I have sometimes wanted it for VS scenarios/binlogs too. I wonder if we should consider logging it in BuildManager instead.

(This isn't blocking: it's a huge improvement as it is)